azurelinuxagent/ga/update.py

```diff
@@ -670,7 +670,7 @@ def _upgrade_available(self, protocol, base_version=CURRENT_VERSION):
             return False
 
         family = conf.get_autoupdate_gafamily()
-        logger.verbose("Checking for agent family {0} updates", family)
+        logger.info("Checking for agent updates (family: {0})", family)
```

Comment:
NIT: Not sure how helpful this would be, but we could also log the `self.last_attempt_time` to analyze how much it drifts from the 1hr pattern

Reply:
since now the message is always there one could just compare the timestamps of the messages themselves. good enough?